twisted/spread/jelly.py

```diff
@@ -88,12 +90,15 @@
         # sets module is deprecated in Python 2.6, and gone in
         # Python 3
         _sets = None
+    else:
+        _SetTypes.append(_sets.Set)
```

Comment:
minor comment: can these code raise an ImportError, why not move it closer to the `import sets` ... I find it easier to read